src/errors/validation-error.ts

```diff
@@ -3,14 +3,14 @@ type ErrorType = 'parser-error' | 'invalid-file' | 'no-spec-found';
 interface IValidationErrorInput {
   type: ErrorType;
   err?: any,
-  filepath?: string,
+  filepath?: any,
```

Comment:
```suggestion
  filePath?: string,
```

Reply:
`filePath` means refactoring multiple files, so instead of simple 2 files PR we get additional 3 unrelated files changed.

Reply:
Aaa ok...